libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/RtspMediaTrack.java

```diff
@@ -121,6 +123,16 @@ public int hashCode() {
         checkArgument(!fmtpParameters.isEmpty());
         processAacFmtpAttribute(formatBuilder, fmtpParameters, channelCount, clockRate);
         break;
+      case MimeTypes.AUDIO_AMR_NB:
+      case MimeTypes.AUDIO_AMR_WB:
+        checkArgument(channelCount == 1, "multi channel is not supported currently");
```

Comment:
Is multichannel not supported because

- AMR doesn't support multi-channel
- RTP AMR doesn't support multi-channel
- The current implementation doesn't support multi-channel?

ALso for L131 and L133.

Reply:
AMR as a codec itself doesn't support multi-channel. However, a reader can send multi-channel input by adding extra handling on its end.
As our implementation doesn't have code for these handling that is why we are not supporting multichannel currently.

Reply:
Acknowledged.